netbox/netbox/api/serializers/fields.py

```diff
@@ -1,5 +1,8 @@
+from django.utils.translation import gettext_lazy as _
```

Comment:
Just FYI, this is to fix an error my IDE was showing that we were using `_` without importing it